block/src/Pos/GState/BlockExtra.hs

```diff
@@ -177,26 +175,6 @@ loadBlocksUpWhile
     -> m (OldestFirst [] Block)
 loadBlocksUpWhile = loadUpWhile getBlock
 
--- | Produce blocks from a database (MonadBlockDBRead) beginning at a hash.
--- Forward links are resolved after each block is yielded, and the source
--- ends when that forward link does not determine another block.
--- Semantics depend upon the MonadBlockDBRead instance, of course. If the
--- underlying database is changed while streaming, what happens?
-blocksSourceFrom
-    :: ( MonadBlockDBRead m )
-    => HeaderHash -> ConduitT () Block m ()
-blocksSourceFrom fromH = do
-    mBlk <- lift $ getBlock fromH
-    case mBlk of
-        Nothing  -> pure ()
-        Just blk -> yield blk >> continue blk
-  where
-    continue blk = do
-        mNextH <- lift $ resolveForwardLink blk
-        case mNextH of
-            Nothing    -> pure ()
-            Just nextH -> blocksSourceFrom nextH
```

Comment:
@karknu relevant to what we were talking about earlier.

Reply:
It's deleted here because it's not used yet, but you will probably want to add something similar. Pipe or conduit or similar, whatever you think is appropriate

Reply:
Me? Or @karknu ? :)

I'll explain again: i have a vision^{tm} of how conduits should be structured. Namely, all traversals should be conduit. And when they are, the particular deleted function will become redundant. Also it wasn't used and it's always available in git history, just in case.

Reply:
Nope, I meant @karknu . We were discussing block streaming earlier today.